asdf_astropy/converters/fits/tests/test_fitswcs.py

```diff
@@ -0,0 +1,31 @@
+import warnings
+
+import asdf
+import pytest
+from astropy.io import fits
+from astropy.utils.data import download_file
+from astropy.wcs import WCS, FITSFixedWarning
+
+
+def create_wcs():
+    urls = [
+        "http://data.astropy.org/tutorials/FITS-cubes/reduced_TAN_C14.fits",
```

Comment:
This would introduce downloading data files as part of the test runs. This is not something done by other tests and I don't think this belongs in unit tests.

Would you add a fixture that creates suitable wcs objects?

Reply:
Please see [this](https://github.com/astropy/asdf-astropy/pull/235#discussion_r1734940367)